hiredis-client/ext/redis_client/hiredis/hiredis_connection.c

```diff
@@ -150,6 +150,12 @@ static void *reply_append(const redisReadTask *task, VALUE value) {
 }
 
 static void *reply_create_string(const redisReadTask *task, char *cstr, size_t len) {
+    if (len > 4 && task->type == REDIS_REPLY_VERB) {
```

Comment:
Might want a `>= 4` here to handle empty verbatim strings: `"txt:"`.

Reply:
- [ ] greater than or equal to

Reply:
fixed: b97892d